inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/service/ConceptLinkingService.java

```diff
@@ -122,6 +122,12 @@ public void init()
         candidateCache = Collections.synchronizedMap(new LRUCache<>(properties.getCacheSize()));
         semanticSignatureCache = Collections
             .synchronizedMap(new LRUCache<>(properties.getCacheSize()));
+
+        MENTION_CONTEXT_SIZE = properties.getMentionContextSize();
```

Comment:
Capitalized variable names are reserved for constants. Better fetch the values always directly from the properties instead of storing them in variables.